demo/docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - tmail
 
   tmail-backend:
-    image: linagora/tmail-backend:memory-branch-master
+    image: chibenwa/tmail-backend:23072024-1
```

Comment:
I understand it's likely for CI tests, but dont forget to put it back to its normal value